src/main/java/design/MazeDesignerInteractor.java

```diff
@@ -0,0 +1,47 @@
+package design;
+
+import entities.DesignableMaze;
+
+public class MazeDesignerInteractor implements MazeDesignerInputBoundary {
+
+    DesignableMaze dm;
+
```

Comment:
I highly suggest adding a constructor to this interactor. 

This constructor can take in an instance of a DesignableMaze and an instance of the outputBoundary:

```
public MazeDesignerInteractor(DesignableMaze dm, MazeDesignerOutputBoundary ob) {
   // update designablemaze instance attribute
   this.dm = dm;
   // update the randomizedprim instance attribute 
   this.mg = new RandomizedPrim(dm);
   // update the output boundary instance attribute
   this.ob = ob;
} 
```